automol/embed/_cleanup.py

```diff
@@ -402,8 +434,8 @@ def _is_converged(xmat, err, grad):
 
 
 def distance_convergence_checker_(
-    lmat: NDArrayLike2D, umat: NDArrayLike2D, max_dist_err=0.2
-):
+    lmat: NDArrayLike2D, umat: NDArrayLike2D, max_dist_err: float = 0.2
+) -> numpy.ndarray:
```

Comment:
This returns a callable like the other convergence checkers (can copy the type over).